### ISSUE TITLE
Broken URL in Project Coala

### ISSUE THREAD

**Issue by foivospro** [MunGell/awesome-for-beginners#1398]
I have noticed an issue with the URL in the Coala project. The current URL redirects to the project's issues.

**Comment by TaiTaiTai777** [comment]
It sounds like you're having an issue with the URL link in the Coala project, where instead of pointing to the intended destination, it redirects to the project's issues page. This can be problematic, especially if the link is supposed to direct users to documentation, a repository, or another pertinent resource. Here’s a step-by-step guide on how to address this issue:

### Steps to Identify and Fix the URL Redirection Issue

1. **Identify the Problematic URL:**
   - Locate the specific part of the Coala project where the incorrect URL is used. This could be in the README file, documentation, or any other part of the project.
   - Identify if the URL is hardcoded, dynamically generated, or retrieved from a configuration file.

2. **Understand the Intended Destination:**
   - Determine where the URL is supposed to lead. Common destinations might include the project’s main repository, documentation site, or a specific page within the GitHub repository.

3. **Check the Redirect:**
   - Open the problematic URL in a web browser or use a tool like `curl` to see where it redirects.
   - For example, running the command below in a terminal can help you trace the redirection path:
     ```sh
     curl -I <problematic-URL>
     ```

4. **Locate the Correct URL:**
   - If the redirection is unintended, find the correct URL by navigating to the intended destination manually. Make sure you copy the correct and updated URL.

5. **Update the URL in the Project:**
   - Edit the files containing the incorrect URL and replace it with the correct one. This might involve updating the README file, documentation, or other relevant files.
   - Ensure consistency by checking for the incorrect URL throughout the project.

6. **Test the Fix:**
   - After updating the URL, test it to ensure it now points to the correct destination without redirection.
   - Consider reviewing the changes with other team members or contributors if available.

7. **Commit and Push Changes:**
   - Once verified, commit the changes with a descriptive commit message. For instance:
     ```sh
     git add <modified-files>
     git commit -m "Fix the URL to point to the correct destination instead of issues page"
     git push origin <branch-name>
     ```

8. **Create a Pull Request:**
   - If you’re working on a fork or a separate branch, create a pull request (PR) to merge the changes into the main branch.
   - Provide a detailed description of the issue and how you fixed it.

### Example Commit Message
Here is an example of a commit message for addressing this issue:
```
Fix: Update URL to direct to the correct resource instead of issues page

The URL in the README.md was redirecting to the project's issues page. 
This update corrects the URL to point to the intended documentation page.

Closes #123 (Replace with relevant issue number if applicable)
```

By following these steps, you can ensure that the URLs in your project are correctly pointing to their intended destinations, improving the user experience and accessibility of resources within the Coala project.

**Comment by sammyhori** [comment]
Fixed in #1424
